app/Models/FeedDAO.php

```diff
@@ -421,4 +423,22 @@ public static function daoToFeed($listDAO, $catID = null) {
 
 		return $list;
 	}
+
+	public function updateTTL() {
+		$sql = <<<SQL
+UPDATE `{$this->prefix}feed`
+   SET ttl = :ttl_default
+ WHERE ttl = -2
+SQL;
+		$stm = $this->bd->prepare($sql);
+		$stm->execute(array(':ttl_default' => FreshRSS_Feed::TTL_DEFAULT));
+
+		$sql = <<<SQL
+UPDATE `{$this->prefix}feed`
+   SET ttl = -900
```

Comment:
When we "unmute", will that feed then be set to `+900`? (That seems too fast)
Maybe we could keep this `-1`? This relates to another comment higher up: isn't there a need to have a value for muted when it was zero. (I have not completely though about the logic yet)

Reply:
It behave the way you understood. It might be too fast, but I can change the value for something more suitable. I don't like having the ```-1``` option because it means introducing a different behavior for that value. We already have a different behavior for the default value.

Reply:
Ok, then let's change it to `TTL_DEFAULT` then.

Reply:
I've changed the value to ```1200```. Think that value is the default value when you make a fresh install of FRSS